app/backend/src/tests/test_search.py

```diff
@@ -73,3 +73,29 @@ def test_regression_search_in_area(db):
             )
         )
         assert [result.user.user_id for result in res.results] == [user3.id, user4.id]
+
+
+def test_user_filter_complete_profile(db):
+    """
+    Make sure the completed profile flag returns only completed user profile
+    """
+
+    user_complete_profile, token6 = generate_user(about_me="this about me is not empty", avatar_key="test_avatar_key_1")
+
+    user_incomplete_profile, token7 = generate_user(about_me=None, avatar_key="test_avatar_key_2")
+
+    with search_session(token7) as api:
+        res = api.UserSearch(
```

Comment:
It's complaining about formatting here. These `res = ` lines should be one-liners.